### PR TITLE
Large scale change: pre-commit checks vol9

### PR DESCRIPTION
**Pull request by kaidokert**
Runs pre-commit run --all and fixes formatting in a number of starboard files. This is according to current linter+formatter standards.

b/278726440
b/199280707

### CODE REVIEW

**Comment by kaidokert** [comment]
Prior PR's in this series #1010 #1011 #1012 #1014 #1015 #1040 #1046

**Comment by andrewsavage1** [comment]
Please see if https://github.com/youtube/cobalt/pull/1046 is causing win32 build/test failures before merging this

**Comment by kaidokert** [comment]
> Please see if #1046 is causing win32 build/test failures before merging this

Yep it was, fixed version in #1061

**Review by yuying-y — APPROVED** [review]

File: starboard/android/shared/log_flush.cc [hunk 1]
```diff
@@ -17,5 +17,4 @@
 #include <stdio.h>
 #include <unistd.h>
 
-void SbLogFlush() {
-}
+void SbLogFlush() {}
```
Comment (hlwarriner) [inline starboard/android/shared/log_flush.cc:20]: The most satisfying change in this PR, IMO.

**Review by hlwarriner — APPROVED** [review]

File: starboard/android/shared/log_flush.cc (hunk 1 above)
Comment (kaidokert) [reply starboard/android/shared/log_flush.cc:20]: We do what we can :)

**Pull request merged**